src/python/gem5/components/processors/simpoint.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2021 The Regents of the University of California
```

Comment:
I'm not sure this file makes sense in the "processors" package. Maybe in `util`? @BobbyRBruce, any opinion?

Reply:
Yeah. I don't think this file has a logical home, but i think `util` makes the most sense here.


Also, the copyright should be 2022, not 2021.